### DISCUSSION TITLE
Version

### DISCUSSION THREAD

**Discussion by nevzs** [imsyy/home#37] [Announcements]
Need english version pls, i cant read @.@

**Comment by imsyy** [comment]
Understand, I will add i18 support in the future after the VUE refactoring 😜

**Comment by albaz64** [comment]
> Understand, I will add i18 support in the future after the VUE refactoring 😜

Maybe you can take a look at my PHP version. Although the comments are in Chinese, there are not many words. It can be easily understood using a translation site (probably

https://github.com/XYZ-dragon/home

I assign values to most variable elements in the form of key-value pairs
